frontend/website/pages/docs/tokens.mdx

```diff
@@ -0,0 +1,98 @@
+
+When creating a new token, a new token ID is created as well as a token account public key. This token account owns the token and can mint more of the token. The account will also (eventually) have the power to enable/disable accounts, and to set whether new accounts may be created for the token.
+
+Please note that creating new tokens requires an additional fee on top of the ever-present transaction fee. This additional fee is the fee that must be paid when creating a new token account. This means there is a fee for the account creation on top of the transaction fee that is normally applied.
+
+### Create a new token
+
+```
+$ coda client create-token -sender <PUBLIC_KEY>
+```
+
+<Alert>
+
+The required field for creating a new token account is: 
+
+- sender : The public key from which you want to send the transaction
+
+</Alert>
+
+See 
+```
+$ coda client create-token -help
+```
+to learn more about how to use the command.
+
+## Creating Token Accounts
+
+With the CLI, you are able to create a token account for existing tokens. Token accounts are like regular accounts but solely meant to interact with the non-default tokens that exist in the Coda protocol. You must create a separate unique token account for each type of token you wish to interact with.
+<Alert kind="warning">
+
+To send or receive an existing token, you must have a token account with the specified token. If there is no valid token account with the token, the transaction will fail
+
+</Alert> 
+
+### Create a new token account 
+
+```
+$ coda client create-token-account -token-owner <PUBLICKEY> -receiver <PUBLICKEY> -sender <KEY> -token <TOKEN_ID>
+```
+
+<Alert>
+
+ If the token owner is not given, the CLI will query to find the token owner. If no owner is found, the command will fail with an error.
+
+The required fields for creating a new token account are:
+
+-receiver : The public key to create the new account for
+
+-sender : The public key from which you want to send the transaction
+
+-token : The ID of the token from which you want to send the transaction
+
+</Alert> 
+
+See 
+```
+$ coda client create-token-account -help
+```
+to learn more about how to use the command.
+
+## Minting Tokens
+
+With the CLI, you are able to mint your own tokens! Minting your own tokens simply increases the supply of a particular token. Minting tokens is how all non-default tokens are created, there are no other protocol-events or commands that can create non-default tokens. This command may only be issued by the token owner, but the tokens may be minted in any existing account for that token.
+
+
+### Mint a token
+
+```
+$ coda client mint-tokens -amount <VALUE> -sender <PUBLIC-KEY> -token <TOKEN_ID>
```

Comment:
Since it was mentioned in the blurb above, might it be worth mentioning the optional `-reveiver` parameter?